wgpu-native/src/resource.rs

```diff
@@ -280,6 +280,16 @@ pub enum CompareFunction {
     Always = 7,
 }
 
+impl CompareFunction {
+    pub fn is_trivial(&self) -> bool {
```

Comment:
nit: `uses_value` or similar might be more descriptive

Reply:
but then you'd have to specify which value is being used :)